_posts/2020-03-22-worldwide-rse-mentoring-network.md

```diff
@@ -0,0 +1,240 @@
+some beginning scripts and code, but doesn't have a good handle on what packages
+to use, and how best to make all those packages and libraries work together.
+How do you and I help **that** person?
+
+I can think of a couple of solutions for this. I have a feeling that people
+that are much smarter than me have already have spent a lot of time
+working on solutions for this. But since I am a scientist,
+I enjoy finding out how wrong I am. So here I will outline two straw man
+solutions, and I hope people will comment on them.
+
+### straw man solution number 1: the Carefully-Curated Collection of Domain-Specific Scientific Python Projects
+
+The first straw man solution is to provide newbie scientist-coders with a
+collection of Python projects, specific to their domain, carefully curated by
+people with more experience.
+You might ask, justifiably, "isn't that what PyOpenSci is?"
+If a project goes through the PyOpenSci process, isn't it kind of *by definition*
+a carefully-curated domain-specific scientific Python project?
+I think the group as a whole is still open to different answers, but if we are
+following our own
+[guidelines about what in scope](https://www.pyopensci.org/dev_guide/peer_review/aims_scope.html),
+and the model of ROpenSci,
+then our group is more focused on building tooling that helps you,
+the scientist with domain expertise, get your job done, without having to
+reinvent the wheel.
+
+And, yeah, as we continue to build that tooling, there will be more and more
+projects that make use of it, and we could in some automated way see all the
+GitHub repos linked with papers that use `DomainToolX`. Again, if we are
+following the lead of ROpenSci, we'd have something like
+[`makeDepGraph`](https://www.rdocumentation.org/packages/miniCRAN/versions/0.2.12/topics/makeDepGraph)
+But we are not there yet.
+
+You might also say, justifiably, aren't these curated collections
+springing up organically? Sure, *sort of*.
+I mean, don't I go to
+[Ye Olde Gallery Of Interesting Notebooks](https://github.com/jupyter/jupyter/wiki/A-gallery-of-interesting-Jupyter-Notebooks)
+every time I give an intro talk, and look for something field specific that
+I hope will catch people's interest?
+But again, I think this is just the *start* of giving someone that is
+relatively new to writing research code an idea of how to put their project
+together. I want there to be so many examples that I could really crunch some
+numbers on them. I want to be able to calculate a meaningful statistic about
+the number of papers on topic $a$ using function $b$ from library $c$.
+
+So the straw man solution is to start some group that curates a
+collection of scientific Python projects.
+It bothered me so much that I didn't have a good answer for that student that
+asked for examples that I made an attempt at this, that you can see in the
+PyOpenSci discourse here:
+<https://pyopensci.discourse.group/t/real-world-repo-packs-looking-for-examples/151>
```

Comment:
another example of a link that won't work.

Reply:
Not sure why this one won't work? Please see reply below